ui/pages/first-time-flow/onboarding-initiator-util.js

```diff
@@ -28,29 +13,22 @@ const returnToOnboardingInitiatorTab = async (onboardingInitiator) => {
     log.warn(
       `Setting current tab to onboarding initiator has failed; falling back to redirect`,
     );
+
+    if (browser.runtime.lastError) {
```

Comment:
I thought the polyfill checked `lastError` for us. This was part of how errors were thrown in the Chrome API. I would expect the pollyfill to check this and reject the call with the error if it's present.

Reply:
Yeah it sounds like you're right:

from https://github.com/mozilla/webextension-polyfill:

>Rather than checking the chrome.runtime.lastError property from every callback, code which needs to explicitly deal with errors registers a separate Promise rejection handler.

Reply:
updated here: https://github.com/MetaMask/metamask-extension/pull/14550/commits/eb9504738b1b35b383d5c9c2bce708fae2b1cfa4